certbot/certbot/_internal/cli/__init__.py

```diff
@@ -95,6 +95,11 @@ def prepare_and_parse_args(plugins: plugins_disco.PluginsRegistry, args: List[st
         action="store_true", default=flag_default("preconfigured_renewal"),
         help=argparse.SUPPRESS
     )
+    helpful.add(
```

Comment:
One very minor thing: would you be able to move this entry lower, maybe below `--preferred-challenges`? 

`argparse` lists flags in the order they are added, and I think we should try keep the more important flags at the top.